src/services/UpdateTechRecordService.ts

```diff
@@ -3,7 +3,7 @@ import {Configuration} from "../utils/Configuration";
 import {HTTPMethods} from "../models/enum";
 
 export class UpdateTechRecordService {
-    public static updateStatusBySystemNumber(systemNumber: string, testStatus: string, testResult: string, testTypeId: number, newStatus: string) {
+    public static updateStatusBySystemNumber(systemNumber: string, testStatus: string, testResult: string, testTypeId: number, newStatus: string, createdById: string, createdByName: string) {
```

Comment:
Did you make sure that all the places where this service was called previously are passing these new fields? Or you just added them to the calls as part of 10316?

Reply:
these fields were added as part of 10316 and I updated the tests and mocks where necessary